docs/core/toc.yml

```diff
@@ -85,7 +85,7 @@
       href: tutorials/cli-templates-create-project-template.md
     - name: 3 - Create a template pack
       href: tutorials/cli-templates-create-template-pack.md
-  - name: Get started with .NET Core on macOS
+  - name: Get started with .NET Core on macOS using Visual Studio Code
```

Comment:
The change looks good but I think we should shuffle some topics around in the TOC later

This one probably belongs in the Get Started node. The ones I'm working on are a sequence of tutorials that probably should be in the tutorials node with a well-defined sequence

Reply:
Do you want me to change it now?

Reply:
If we can, sure! Otherwise, we can do it later.